desktop/core/src/desktop/js/apps/storageBrowser/StorageBrowserPage/StorageBrowserTable/StorageBrowserTable.tsx

```diff
@@ -195,14 +194,15 @@ const StorageBrowserTable = ({
       };
       if (key === 'name') {
         column.width = '40%';
-        //TODO: Apply tooltip only for truncated values
         column.render = (_, record: StorageBrowserTableData) => (
-          <Tooltip title={record.name}>
+          <>
             <span className="hue-storage-browser__table-cell-icon">
               {record.type === 'dir' ? <FolderIcon /> : <FileOutlined />}
             </span>
-            <span className="hue-storage-browser__table-cell-name">{record.name}</span>
-          </Tooltip>
+            <span className="hue-storage-browser__table-cell-name" title={record.name}>
```

Comment:
The main objective to remove tooltip was to avoid getting tooltip even for short text. Even when the full text is visible then also we are rendering the tooltip, which somehow duplicate the information on the screen. By adding title, it brings the 2 sec delay in getting a default tooltip.
Now I am thinking to introduce similar delay in tooltip or conditionally render the tooltip for longer texts.

What are your thoughts?

Reply:
I also think adding a delay make sense. If there is a delay the duplication is not a problem in my opinion.

Reply:
I have added the delay in existing tooltip, and removed the title tag.